x/ccv/provider/keeper/keeper.go

```diff
@@ -683,11 +683,13 @@ func (k Keeper) DeleteInitChainHeight(ctx sdk.Context, chainID string) {
 }
 
 // GetPendingVSCs returns the list of pending ValidatorSetChange packets stored under chain ID
-func (k Keeper) GetPendingVSCs(ctx sdk.Context, chainID string) (packets []ccv.ValidatorSetChangePacketData, found bool) {
+func (k Keeper) GetPendingVSCs(ctx sdk.Context, chainID string) []ccv.ValidatorSetChangePacketData {
+	var packets []ccv.ValidatorSetChangePacketData
```

Comment:
This is equiv to `packets := []ccv.ValidatorSetChangePacketData{}` right? Just confirming the slice gets initialized to empty with var syntax

Reply:
Behaviour is equivalent.

Technically it's a nil slice but all operations on it are permitted and will not panic unless you try to access an index.

It is safe to iterate over and append to it